README.md

```diff
@@ -61,6 +61,18 @@ Build binary:
 make build
 ```
 
+Run Unit/Integ Tests:
+```console
+make test
+```
+(see internal/README.md for explanation of tests)
+
+Run tests against kind cluster and live controller:
+```console
+UPTEST_EXAMPLE_LIST=$(find examples/*.yaml -type f| tr '\n' ',') make e2e
```

Comment:
Would it make sense to add this to the e2e or uptest make step so it automatically happens on invoc?

Reply:
@Monkeyanator yes, we are doing similar stuff in our platform-refs https://github.com/upbound/platform-ref-azure/blob/main/Makefile#L62 and just testing the specific static set of examples in e2e make target. So please feel free to adjust make target to the most convenient scenario for your provider, e.g. just testing all examples with the same target.